BillTrack.Persistence/Repositories/ICrudRepository.cs

```diff
@@ -0,0 +1,10 @@
+namespace BillTrack.Persistence.Repositories;
+
+interface ICrudRepository<T>
```

Comment:
CRUD could be extended later. There is no point in adding "CRUD" to the name.

Please, update the name to a simple `IRepository<T>`.

Reply:
fixed